operatorcourier/identify.py

```diff
@@ -20,17 +20,14 @@ def get_operator_artifact_type(operatorArtifactString):
         logger.error(msg)
         raise OpCourierBadYaml(msg)
     else:
-        artifact_type, artifact_name = None, None
+        artifact_type = None
```

Comment:
This change seems unrelated to the purpose of the pr. I agree that it is a valid change, but it should be separated into a separate pr.

Reply:
Thanks @kevinrizza. Removed the change and created a PR https://github.com/operator-framework/operator-courier/pull/123 for this change.